package.json

```diff
@@ -25,8 +25,10 @@
     "@internetarchive/feature-feedback": "^0.1.2",
     "@internetarchive/field-parsers": "^0.1.3",
     "@internetarchive/histogram-date-range": "^0.1.6",
+    "@internetarchive/ia-activity-indicator": "0.0.3",
```

Comment:
```suggestion
    "@internetarchive/ia-activity-indicator": "^0.0.3",
```

caret

Reply:
hi Neeraj, note that we have been having issues with stale lock files, please update lock files as well in final PR